src/containers/modals/about-modal/teamMembers.jsx

```diff
@@ -7,43 +7,52 @@ const teams = [
+      { name: 'Kiara Cobb', title: 'Development Coordinator' },
     ],
   },
   {
     team: 'Map of Life',
     members: [
-      { name: 'Alexander Killion', title: 'Managing Director' },
-      { name: 'Yanina Sica', title: 'Biodiversity Data Specialist' },
-      { name: 'Melissa Slater', title: 'ESRI Biodiversity Data Specialist' },
-      { name: 'John Wilshire', title: 'Software Engineer' },
-      { name: 'Jeremy Cohen', title: 'Associate Research Scientist' },
-      { name: 'Kevin Winner', title: 'Modeling Project Lead' },
-      { name: 'Kalkidan Chefira', title: 'Software Engineer' },
-      { name: 'Julia Portmann', title: 'Research Associate' },
-      { name: 'Tamara Rudic', title: 'Science Communication Specialist' },
-      { name: 'Danyan Leng', title: 'Postgraduate Associate' },
-      { name: 'Christopher Manciero', title: 'Senior Software Developer' },
+      { name: 'Walter Jetz ', title: 'Lead PI of Map of Life – planning and oversight of execution of project activities; supervision of research and software engineering staff' },
+      { name: 'Alexander Killion', title: 'Managing Director – Geospatial prioritization and conservation' },
+      { name: 'John Wilshire', title: 'Software Engineer - Software engineering' },
+      { name: 'Christopher Manciero', title: 'Sr. Software Developer - Software engineering' },
+      { name: 'Kalkidan Chefira', title: 'Software Engineer - Software engineering' },
+      { name: 'Beth Gerstner', title: 'Research Associate - Spatial analytics' },
+      { name: 'Melissa Slater', title: 'Data Associate – Data management' },
+      { name: 'Kevin Winner', title: 'Spatial Modelling Lead - Spatial analytics' },
+      { name: 'Julia Portmann', title: 'Research Associate - Spatial analytics' },
+      { name: 'Tamara Rudic', title: 'Science Communication Specialist – StoryMaps and UI/UX' },
+      { name: 'Danyan Leng', title: 'Postgraduate Associate – Conservation science and spatial data' },
+      { name: 'Jeremy Cohen', title: 'Associate Research Scientist, Biodiversity and Climate change' },
+      { name: 'Claire Hoffmann', title: 'Science and User Partner Engagement' },
+      { name: 'Frank La Sorte', title: 'Consultant, Spatial biodiversity and Global Change' },
+      { name: 'Meredith Palmer', title: 'Biodiversity data integration and change analytics ' },
     ],
   },
   {
     team: 'Vizzuality',
     members: [
-      { name: 'Craig Mills', title: 'CEO of Vizzuality' },
-      { name: 'Susana Romao', title: 'Project Manager' },
-      { name: 'Sofía Aldabet', title: 'Scientist' },
+      { name: 'David Gonzalez', title: 'CEO of Vizzuality' },
+      { name: 'Susana Romao', title: 'Project Portfolio & Partnerships Lead' },
+      { name: 'Sofía Aldabet', title: 'Data Management' },
+      { name: 'Ángel Arcones', title: 'Data Management' },
       { name: 'Álvaro Leal', title: 'Full-stack Developer' },
+      { name: 'Ana Montiaga', title: 'Full-stack Developer' },
```

Comment:
Ana has evolved to a PM in Vizzuality. Is no longer a developer. Could you remove her from the list?